src/System.Private.Reflection.Core/src/System/Reflection/Runtime/MethodInfos/RuntimeConstructorInfo.cs

```diff
@@ -105,7 +105,7 @@ public sealed override object Invoke(object obj, BindingFlags invokeAttr, Binder
             return methodInvoker.Invoke(obj, parameters);
         }
 
-        public sealed override int MetadataToken
+        public override int MetadataToken
```

Comment:
I think it'd be better to move this to a sealed override in RuntimeSyntheticConstructorInfo and make the override in RuntimeConstructorInfo an "abstract override".